src/renderer/components/dex/MarketSelection.vue

```diff
@@ -71,16 +73,65 @@ export default {
       return this.storeStateCurrentMarket && this.$store.state.holdings.length ?
         this.$services.neo.getHolding(this.storeStateCurrentMarket.baseAssetId).unitValue : 0;
     },
+
+    close24Hour() {
+      return this.todayTrades.length > 0 ?
+        this.todayTrades[0].price : 0;
+    },
+
+    open24Hour() {
+      return this.todayTrades.length > 0 ?
+        this.todayTrades[this.todayTrades.length - 1].price : 0;
+    },
+
+    low24Hour() {
+      return this.todayTrades.length > 0 ?
+        _.minBy(this.todayTrades, trade => Number(trade.price)).price : 0;
+    },
+
+    high24Hour() {
+      return this.todayTrades.length > 0 ?
+        _.maxBy(this.todayTrades, trade => Number(trade.price)).price : 0;
+    },
+
+    volume24Hour() {
+      return this.todayTrades.length > 0 ?
+        _.sumBy(this.todayTrades, trade => Number(trade.quantity)) : 0;
+    },
+
+    change24HourPercent() {
+      return this.todayTrades.length > 0 ?
+        Math.round(
+          ((this.close24Hour - this.open24Hour) / this.open24Hour) * 10000) / 100 : 0;
+    },
+
+    change24Hour() {
+      return this.close24Hour - this.open24Hour;
+    },
+
     percentChangeAbsolute() {
-      return this.$store.state.tradeHistory ?
-        Math.abs(this.$store.state.tradeHistory.change24HourPercent) : 0;
+      return Math.abs(this.change24HourPercent);
+    },
+
+    todayTrades() {
+      return this.trades().length > 0 ? _.filter(this.trades(),
```

Comment:
It probably is better not to have the view be keeping a copy of these. Lets talk offline about how this data should get populated potentially another backend API.